ocpp/v16/schemas/StopTransaction.json

```diff
@@ -1,166 +1,175 @@
+                                        "Power.Offered",
+                                        "Power.Reactive.Export",
+                                        "Power.Reactive.Import",
+                                        "Power.Factor",
+                                        "Current.Import",
+                                        "Current.Export",
+                                        "Current.Offered",
+                                        "Voltage",
+                                        "Frequency",
+                                        "Temperature",
+                                        "SoC",
+                                        "RPM"
+                                    ]
+                                },
+                                "phase": {
+                                    "type": "string",
+                                    "additionalProperties": false,
+                                    "enum": [
+                                        "L1",
+                                        "L2",
+                                        "L3",
+                                        "N",
+                                        "L1-N",
+                                        "L2-N",
+                                        "L3-N",
+                                        "L1-L2",
+                                        "L2-L3",
+                                        "L3-L1"
+                                    ]
+                                },
+                                "location": {
+                                    "type": "string",
+                                    "additionalProperties": false,
+                                    "enum": [
+                                        "Cable",
+                                        "EV",
+                                        "Inlet",
+                                        "Outlet",
+                                        "Body"
+                                    ]
+                                },
+                                "unit": {
+                                    "type": "string",
+                                    "additionalProperties": false,
+                                    "enum": [
+                                        "Wh",
+                                        "kWh",
+                                        "varh",
+                                        "kvarh",
+                                        "W",
+                                        "kW",
+                                        "VA",
+                                        "kVA",
+                                        "var",
+                                        "kvar",
+                                        "A",
+                                        "V",
+                                        "K",
+                                        "Celcius",
```

Comment:
It seems that `Celsius` is missing here.

Reply:
That's right.
I'll bring it back to OCA, as I prefer to not touch the schemas here. (I think it's better to use the upstream schemas if possible).

Reply:
Fair point.
I guess that the OCA fix the bug in a new V5 errate. We can implement the fix than after the errata has been released.

For now I am ready to merge the PR. Thanks for your help!

Reply:
I also have contacted the OCA. It seems that they on purpose want the schemas to remain unchanged and that, for now, CS must adapt the enumerations and include both spellings and CP must correct the spelling.